Basic/Primitive/primitive.pd

```diff
@@ -2179,9 +2179,45 @@ As of 2.062, the generator changed from Perl's generator to xoshiro256++
 
 =for usage
 
+From within pdl:
+
  srand(); # uses current time
  srand(5); # fixed number e.g. for testing
 
+This  code above will not seed PDL's generator, but Perl's rand in Perl.  
```

Comment:
I asked you to remove the code that doesn't work. Changing it to code that *does* work, and adding why it needs to be that, is fine. But you're leaving it with the first thing there as something that doesn't work, which is very unhelpful.

Please change this. It won't get merged until that's done.